sdk/helper/certutil/helpers.go

```diff
@@ -804,3 +805,60 @@ func SignCertificate(data *CreationBundle) (*ParsedCertBundle, error) {
 
 	return result, nil
 }
+
+func NewCertPool(pathToFile string) (*x509.CertPool, error) {
```

Comment:
Consider having this and "certsFromFile" take an `io.Reader` instead. This would decouple it from the source details make it easy to use against files, or strings (e.g. with `strings.NewReader()`), etc. For example, all of the tmpfile code in certutil_test.go could be removed. The whole `certsFromFile()` function might even go away (though I've not looked to see if it's needed elsewhere).

Reply:
That ended up making the code nicer and more reusable. https://github.com/hashicorp/vault/pull/8249/commits/f38381ef5e8f1cabb3768ef1d5d3ff935c5f5ffb